### PR TITLE
Fix(#14560): fix breadcrumb not getting updating properly when clicking on any breadcrumb link in container page

### LINKED ISSUE #14560: Container: The breadcrumb is not getting updated
The breadcrumb is not getting updated as we navigate through the hierarchy of container service.
@open-metadata/ui  needs to check how we are sending values for creating breadcrumb

steps to reproduce:
Go back and forth different levels of hierarchy in containers and click on breadcrumbs to go back.
https://sandbox-beta.open-metadata.org/container/s3_storage_sample.deep_nested_container_1

https://github.com/open-metadata/OpenMetadata/assets/40225091/3ae9b65b-479c-4180-8553-dcc823bbbd58

### PR DESCRIPTION
**Pull request by Abhishek332**
this PR fix #14560 

**### Before fix**

https://github.com/open-metadata/OpenMetadata/assets/80886271/3129c8e0-57a2-4f2c-bbe8-8596f66f1f81

**### After fix**

https://github.com/open-metadata/OpenMetadata/assets/80886271/fb5da5d9-d8c7-4fc6-9763-c385599b1189

### **Fix Skeleton loading on DatabaseDetailsPage also -**
**Before** 

https://github.com/open-metadata/OpenMetadata/assets/80886271/5e0373fa-29ab-4d71-bdae-9e550de9e233

**After**

https://github.com/open-metadata/OpenMetadata/assets/80886271/8a24ab6c-1041-47f0-9549-39e9e3558506

### **Fix LogsViewer Skeleton**
**Before**
Screenshot 2024-01-15 at 9 21 54 PM

**After**
Screenshot 2024-01-15 at 9 21 21 PM

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx [hunk 1]
```diff
@@ -777,6 +777,7 @@ const ContainerPage = () => {
             afterDomainUpdateAction={afterDomainUpdateAction}
             dataAsset={containerData}
             entityType={EntityType.CONTAINER}
+            key={containerData.fullyQualifiedName}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx:780]: @Abhishek332 , can we add the unit test for this?

File: openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.test.tsx [hunk 2]
```diff
@@ -10,240 +10,387 @@
...
-    .fn()
-    .mockReturnValue(<div data-testid="entity-lineage">EntityLineage</div>);
-});
+jest.mock(
+  '../../components/DataAssets/DataAssetsHeader/DataAssetsHeader.component',
+  () => ({
+    DataAssetsHeader: jest.fn().mockImplementation(
+      ({
+        afterDeleteAction,
+        //   afterDomainUpdateAction,
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.test.tsx:124]: Remove the commented code.

File: openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx [hunk 3]
```diff
@@ -152,14 +152,24 @@ const ContainerPage = () => {
...
         ResourceEntity.CONTAINER,
         containerFQN
       );
       setContainerPermissions(entityPermission);
+
+      const viewBasicPermission =
+        entityPermission.ViewAll || entityPermission.ViewBasic;
+
+      if (viewBasicPermission) {
+        await fetchContainerDetail(containerName);
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx:170]:
```suggestion
        await fetchContainerDetail(containerFQN);
```

File: openmetadata-ui/src/main/resources/ui/package.json [hunk 4]
```diff
@@ -17,7 +17,7 @@
     "preinstall": "cd ../../../../.. && yarn install --frozen-lockfile",
     "postinstall": "yarn run build-check",
     "pre-commit": "lint-staged --concurrent false",
-    "test": "jest --passWithNoTests --maxWorkers=3 --silent",
+    "test": "jest --passWithNoTests --maxWorkers=3",
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/package.json:20]: Do not remove this as we don't want the test logs in production environment.

Comment (Abhishek332) [reply openmetadata-ui/src/main/resources/ui/package.json:20]: reverted, accidentally pushed

**Review by Sachin-chaurasiya — DISMISSED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by karanh37 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**